<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chooseWorkerDialogClass</class>
 <widget class="QDialog" name="chooseWorkerDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>chooseWorkerDialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>382</width>
     <height>434</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="refreshButton">
        <property name="text">
         <string>Odśwież</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Sortowanie:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="sortBox">
        <item>
         <property name="text">
          <string>A - Z</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Z - A</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="okButton">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTableWidget" name="workerTable">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <column>
       <property name="text">
        <string>Imię</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nazwisko</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Numer telefonu</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>chooseWorkerDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
